pkg/networkmanager/network-interface-members.jsx

```diff
@@ -0,0 +1,245 @@
+                                    isSmall
+                                    onClick={syn_click(this.model, () => {
+                                        with_checkpoint(
+                                            this.model,
+                                            function () {
+                                                return (free_member_connection(member_con)
+                                                        .fail(show_unexpected_error));
+                                            },
+                                            {
+                                                devices: dev ? [dev] : [],
+                                                fail_text: cockpit.format(_("Removing <b>$0</b> will break the connection to the server, and will make the administration UI unavailable."), iface.Name),
+                                                anyway_text: cockpit.format(_("Remove $0"), iface.Name),
+                                                hack_does_add_or_remove: true
+                                            });
+                                        return false;
+                                    })}>
+                                    <MinusIcon />
+                                </Button>}
+                            </>
+                        )
+                    }
+                ],
+                rowId: iface.Name,
+                extraClasses: this.props.highlight == iface.Name ? ["highlight-ct"] : [],
+                props: {
+                    key: iface.Name,
+                    "data-interface": encodeURIComponent(iface.Name),
+                    "data-sample-id": isActive ? encodeURIComponent(iface.Name) : null
+                }
+            });
+
+            if (isActive) {
+                const samples = this.props.usage_monitor.samples[iface.Name];
+                row.columns.splice(1, 0, { title: samples ? cockpit.format_bits_per_sec(samples[1][0] * 8) : "" });
+                row.columns.splice(2, 0, { title: samples ? cockpit.format_bits_per_sec(samples[0][0] * 8) : "" });
+            } else {
+                row.columns.splice(1, 0, { title: device_state_text(), props: { colSpan: 2 } });
+            }
+
+            rows.push(row);
+        });
+        return rows;
+    }
+
+    render() {
+        const dropdownItems = (
+            this.props.interfaces
+                    .filter(iface => {
+                        return (is_interesting_interface(iface) &&
+                            !this.props.memberIfaces[iface.Name] &&
+                            iface != this.props.iface);
+                    })
+                    .map(iface => {
+                        const onClick = () => {
+                            with_checkpoint(
+                                this.model,
+                                () => {
+                                    const con = iface.MainConnection;
```

Comment:
This needs to be `this.props.iface.MainConnection`.  Otherwise, we would always add the interface to itself.

Reply:
Good catch!

Reply:
Fixed.